libres/lib/res_util/block_fs.cpp

```diff
@@ -864,15 +864,15 @@ static void block_fs_preload(block_fs_type *block_fs) { return; }
 */
 
 static void block_fs_fix_nodes(block_fs_type *block_fs,
-                               long_vector_type *offset_list) {
+                               const std::vector<long>& offset_list) {
     if (block_fs->data_owner) {
         fsync(block_fs->data_fd);
         {
             char *key = NULL;
-            for (int inode = 0; inode < long_vector_size(offset_list);
+            for (int inode = 0; inode < offset_list.size();
```

Comment:
I believe the c tests only call this function with offset_list.size() == 0 and so the brunt of this function only has integration tests.